crates/ruff_notebook/src/notebook.rs

```diff
@@ -145,16 +146,41 @@ impl Notebook {
         // Add cell ids to 4.5+ notebooks if they are missing
         // https://github.com/astral-sh/ruff/issues/6834
         // https://github.com/jupyter/enhancement-proposals/blob/master/62-cell-id/cell-id.md#required-field
+        // https://github.com/jupyter/enhancement-proposals/blob/master/62-cell-id/cell-id.md#questions
         if raw_notebook.nbformat == 4 && raw_notebook.nbformat_minor >= 5 {
+            // We use a mock random number generator to generate deterministic uuids
+            let mut rng = rand::rngs::mock::StepRng::new(0, 1);
```

Comment:
Could we use a seeded number generator rather than the `StepRng` one, so that the IDs are deterministic but look random rather than structured as they do now? Or was this the only option for deterministic UUIDs?

Reply:
I'm sure there's another option for a seeded number generator, this one was just the most obvious way I saw.

Reply:
Sure we can use the `StdRng` seeded with `0`

```
7fb27b94-1602-401d-9154-2211134fc71a
acae54e3-7e7d-407b-bb7b-55eff062a284
9a63283c-baf0-4dbc-ab1f-6479b197f3a8
8dd0d809-2fe7-4a7c-9628-1538738b07e2
72eea511-9410-473a-a328-ad9291626812
```